packages/browser/src/browser/index.ts

```diff
@@ -83,9 +83,16 @@ export function loadLegacySettings(
   const baseUrl = cdnURL ?? getCDN()
 
   return fetch(`${baseUrl}/v1/projects/${writeKey}/settings`)
-    .then((res) => res.json())
+    .then((res) => {
+      if (!res.ok) {
+        return res.text().then((errorResponseMessage) => {
```

Comment:
What does the `text()` end up looking like? In your screenshot it looks like the `statusText` has a useful error message - I wonder if it makes sense to use that instead.

Reply:
I'll chip in because this is something I discussed offline -- that screenshot is obsolete. That text() returns the more detailed message -- it's in the response body. Now the object will be something like:
```
errorResponseMessage: "Cannot GET -- invalid writekey ..."
status: 404
statusText: "Not Found"
```

Reply:
Oh awesome, keep `.text()` then :smile:

Reply:
So that `statusText` is misleading - it wasn't actually the `statusText` that you'd normally get assigned, but rather the one I assigned to `statusText`. It was empty without unwrapping `res.text`.